peachjam/feeds.py

```diff
@@ -85,11 +84,25 @@ class CoreDocumentAtomSiteNewsFeed(CoreDocumentFeed, BaseAtomFeed):
     subtitle = CoreDocumentFeed.description
 
 
+class CustomArticleFeedGenerator(Rss201rev2Feed):
+    def add_item_elements(self, handler, item):
+        super().add_item_elements(handler, item)
+        handler.addQuickElement("image", item.get("image", ""))
+        handler.addQuickElement("body", item.get("body", ""))
+
+
 class ArticleFeed(BaseFeed):
     model = Article
-    title = "Articles"
+    title = "AfricanLII Articles"
```

Comment:
This is in peachjam, not africanlii. So we shouldn't hardcode. What if you pull in the app name from settings.py?